source/guides/views/layouts.md

```diff
@@ -66,6 +66,37 @@ end
 
 If we want to turn off this feature entirely, we can set `layout nil` into the application's configuration.
 
+## Using Multiple Template Layouts
+
+Sometimes it's useful to have more than one layout. For example, if the `application.html.erb` template contains an HTML structure you don't want for, say, a login page - a separate `login.html.erb` template can be used. For example, a `<nav>` element with categories has no place next to a login form.
```

Comment:
What do you think about making this a little simpler? Something like:
```
Sometimes it's useful to have more than one layout. For example, if the `application.html.erb` template contains navigation elements, and you want an entirely different layout, without navigation, for a login page, you can create a `login.html.erb` layout template.
``` 
(feel free to change some wording though!)